README.md

```diff
@@ -207,6 +207,25 @@ Optionally, you can add more command line arguments to the `command` section. Fo
   ```bash
   docker-compose -f /path/to/docker-compose.yml up -d
   ```
+## Running on Google Dataflow
```

Comment:
Give a reference to the "custom commands" section on which we based it (and maybe the sample `setup.py` from their repo?)